docs/dev/table/tableApi.md

```diff
@@ -4062,6 +4075,19 @@ STRING.toBase64()
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight scala %}
+STRING.ltrim()
+{% endhighlight %}
+      </td>
+
+      <td>
+         <p>Returns the result string which trimmed the left spaces from the base string; returns NULL if <i>string</i> is NULL.</p> 
+         <p>E.g., <code>" This is a test String.".ltrim()</code> returns "This is a test String.".</p>
```

Comment:
\<code>" This is a test String.".ltrim()\</code> => \<code>' This is a test String.'.ltrim()\</code>